sections/futures/UserInfo/PositionsTable.tsx

```diff
@@ -0,0 +1,430 @@
+					stopLoss: accountType === 'cross_margin' ? (position as any).stopLoss : undefined,
+					takeProfit: accountType === 'cross_margin' ? (position as any).takeProfit : undefined,
+				};
+			})
+			.filter(
+				({ position, market }) =>
+					position && market && (market?.asset !== currentMarket || showCurrentMarket)
+			);
+	}, [
+		accountType,
+		isolatedPositions,
+		crossMarginPositions,
+		futuresMarkets,
+		positionHistory,
+		currentMarket,
+		t,
+		showCurrentMarket,
+	]);
+
+	return (
+		<div style={{ height: '100%' }}>
+			<DesktopOnlyView>
+				{/* <LegacyLink /> */}
+				<Table
+					data={data}
+					rounded={false}
+					onTableRowClick={(row) =>
+						router.push(ROUTES.Markets.MarketPair(row.original.market.asset, accountType))
+					}
+					noResultsMessage={
+						!isL2 ? (
+							<TableNoResults>
+								{t('common.l2-cta')}
+								<div onClick={switchToL2}>{t('homepage.l2.cta-buttons.switch-l2')}</div>
+							</TableNoResults>
+						) : (
+							<TableNoResults>
+								{!showCurrentMarket ? (
+									t('dashboard.overview.futures-positions-table.no-result')
+								) : (
+									<Link href={ROUTES.Markets.Home(accountType)}>
+										<div>{t('common.perp-cta')}</div>
+									</Link>
+								)}
+							</TableNoResults>
+						)
+					}
+					highlightRowsOnHover
+					columns={[
+						{
+							Header: (
+								<TableHeader>{t('dashboard.overview.futures-positions-table.market')}</TableHeader>
+							),
+							accessor: 'market',
+							Cell: (cellProps: CellProps<typeof data[number]>) => {
+								return (
+									<StyledText>
+										{cellProps.row.original.market.marketName}
+										{/* <MarketBadge
+												currencyKey={cellProps.row.original.market.marketKey}
+												isFuturesMarketClosed={cellProps.row.original.market.isSuspended}
+												futuresClosureReason={cellProps.row.original.market.marketClosureReason}
+											/> */}
+									</StyledText>
+								);
+							},
+							width: 120,
+						},
+						{
+							Header: (
+								<TableHeader>{t('dashboard.overview.futures-positions-table.side')}</TableHeader>
+							),
+							accessor: 'position',
+							Cell: (cellProps: CellProps<typeof data[number]>) => {
+								return <PositionType side={cellProps.row.original.position.side} />;
+							},
+							width: 60,
+						},
+						{
+							Header: (
+								<TableHeader>
+									{t('dashboard.overview.futures-positions-table.notionalValue')}
+								</TableHeader>
+							),
+							accessor: 'notionalValue',
+							Cell: (cellProps: CellProps<typeof data[number]>) => {
+								const formatOptions = cellProps.row.original.position.notionalValue.gte(1e6)
+									? { truncate: true }
+									: {};
+
+								return (
+									<div>
+										<Currency.Price
+											price={cellProps.row.original.position.size}
+											currencyKey={cellProps.row.original.market.asset}
+											showCurrencyKey
```

Comment:
I know it's in the design but I think we can remove the currency key here as it's clear from the first col what asset it is and just causes problem with using up valuable width like so:
<img width="395" alt="Screenshot 2023-04-01 at 09 14 47" src="https://user-images.githubusercontent.com/3802342/229274664-e2266da9-db49-4999-81e9-21f594bed682.png">

cc @asherism

Reply:
Yeah, I agree. It should be easy for users to deduce the currency key, based on the market.